index.html

```diff
@@ -6,59 +6,244 @@
+                what you see and have a project you need coded,
+                don’t hestiate to contact me.
+            </p>
 
-        <ul class="social-media">
-            <li>
-                <a href="">
-                    <img src="./assets/social-media/twitter.svg" alt="twitter icon">
-                </a>
-            </li>
-            <li>
-                <a href="">
-                    <img src="./assets/social-media/linkedin.svg" alt="linkedin icon">
-                </a>
-            </li>
+            <ul class="social-media">
+                <li>
+                    <a href="">
+                        <img src="./assets/social-media/twitter.svg" alt="twitter icon">
+                    </a>
+                </li>
+                <li>
+                    <a href="">
+                        <img src="./assets/social-media/linkedin.svg" alt="linkedin icon">
+                    </a>
+                </li>
+                <li>
+                    <a href="">
+                        <img src="./assets/social-media/medium.svg" alt="medium icon">
+                    </a>
+                </li>
+                <li>
+                    <a href="">
+                        <img src="./assets/social-media/github.svg" alt="github icon">
+                    </a>
+                </li>
+                <li>
+                    <a href="">
+                        <img src="./assets/social-media/angellist.svg" alt="angellist icon">
+                    </a>
+                </li>
+            </ul>
+
+            <button type="button">
+                <img src="./assets/Disabled.svg" alt="disabled button">
+            </button>
+        </div>
+    </section>
+    <section>
+        <div class="works">
+            <h1>My Recent Works</h1>
+            <article class="card-work">
+                <div class="card-image">
+                    <img src="./assets/project1.png" alt="image work">
+                </div>
+                <h2 class="card-title">
+                    Tonic
+                </h2>
+                <ul class="card-tags">
+                    <li>Ruby on rails</li>
+                    <li>css</li>
+                    <li>JavaScript</li>
+                    <li>html</li>
+                </ul>
+                <button type="button">
+                 See Project   
+                </button>
+            </article>
+            <article class="card-work">
+                <div class="card-image">
+                    <img src="./assets/project2.png" alt="image work">
+                </div>
+                <h2 class="card-title">
+                    
+                </h2>
+                <ul class="card-tags">
+                    <li>Ruby on rails</li>
+                    <li>css</li>
+                    <li>JavaScript</li>
+                    <li>html</li>
+                </ul>
+                <button type="button">
```

Comment:
- [ ] Please make sure that the list of technology design match the Figma design 
![image](https://user-images.githubusercontent.com/74744787/159080144-e2c8a7c8-a1bb-4024-820b-bfc240ba8664.png)

![image](https://user-images.githubusercontent.com/74744787/159079298-8a12e4a0-d114-4206-95a2-5d9f48cd184e.png)